Sources/TuistCore/Constants.swift

```diff
@@ -6,6 +6,7 @@ public class Constants {
     public static let binName = "tuist"
     public static let gitRepositoryURL = "https://github.com/tuist/tuist.git"
     public static let version = "0.15.0"
+    public static let envVersion = "0.1.0"
```

Comment:
`tuistenv` and `tuist` are versioned together so you can just use `version`.

Reply:
👍 Makes sense